src/components/filters/useUpdateMethod.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/prop-types */
+import React, { Fragment, useEffect } from 'react';
+import { useUpdateMethodFilter } from './useUpdateMethodFilter';
+import { updateMethodOptions, UPDATE_METHOD_KEY } from '../../Utilities';
+import { mount } from 'enzyme';
```

Comment:
We should avoid `enzyme`. The `@testing-library/react-hooks` is already included in inventory and can be easily used here.

Reply:
@bastilian thank you for having a look. Actually, `enzyme` is used in almost all filter tests, thus I prefer to get rid of it at once later. I would like to refactor the filtering in Inventory, while refactoring I will get rid of enzyme.

Reply:
There is already a ticket to optimise filtering: [RHIF-112](https://issues.redhat.com/browse/RHIF-112)